h/accounts/test/views_test.py

```diff
@@ -47,6 +47,14 @@ def __init__(self, **kwargs):
             setattr(self, k, kwargs[k])
 
 
+class FakeSerializer(object):
+    def dumps(self, obj):
+        return 'faketoken'
+
+    def loads(self, token):
+        return {'username': 'foo@bar.com'}
```

Comment:
Oh, this looks nice! I went for the fake classes since that was the approach that I found already there in the code, but this one looks quite neat. Which one would be the preferred way going forward?